src/bms/lex/token.rs

```diff
@@ -127,7 +129,7 @@ pub enum Token<'a> {
     },
     /// `#MIDIFILE [filename]`. Defines the MIDI file as the BGM. *Deprecated*
     MidiFile(&'a Path),
-    /// Non-empty lines that not starts in # in bms file.
+    /// Non-empty lines that not starts in `['#', '%']` in bms file.
```

Comment:
In fact, the line that starts with `%` is not a command. But `%URL` and `%EMAIL` is currently treated as special tokens reluctantly. We should not recognize other lines such as `%FOO`.